luthier/src/test/groovy/com/yahoo/bard/webservice/config/luthier/FactoryParkTest.groovy

```diff
@@ -0,0 +1,14 @@
+package com.yahoo.bard.webservice.config.luthier
+
+import spock.lang.Specification
+
+class FactoryParkTest extends Specification {
+    void setup() {
+    }
+
+    void cleanup() {
+    }
+
+    def "BuildEntity"() {
```

Comment:
We typically use much longer and more descriptive names, because Spock/Groovy make that easy.
Something like
```groovy
"A FactoryPark<Dimension> correctly builds a Dimension from JSON."() {

```

Reply:
Addressed.